README.md

```diff
@@ -64,6 +64,7 @@ Setting | Required | Type             | Description |
 `sheet_id` | Required | String           | Your target google sheet id
 `output_name` | Optional | String           | Optionailly rename the stream and output file or table from the tap
 `child_sheet_name` | Optional | String           | Optionally choose a different sheet from your Google Sheet file
+`range` | Optional | String | Optionally choose a range of data from your Google Sheet file. Example: `B5:GH` (starts at B5 and goes to columns GH at the end of the sheet)
```

Comment:
Can you clarify here whether start and end cells are inclusive or exclusive, and the behaviour when no range end cell is specified (i.e. `B5:`)?

Reply:
I add a more detailed description. Please look at it, if it is ok for you.

Reply:
Thanks, I'll do a proper review later today hopefully.